### PR TITLE
feature/GH-616

### PR DESCRIPTION
**Pull request by mikelMHybris**
Removed BEM classes
Removed scss variables and bem classes from the scss

### CODE REVIEW

File: projects/storefrontlib/src/lib/product/components/product-list/product-list-item/product-list-item.component.scss [hunk 1]
```diff
@@ -1,73 +1,51 @@
...
+  text-align: var(--cx-text-align, left);
+  display: var(--cx-display, inline-block);
+  margin: var(--cx-margin, 0 0 25px 0);
 
-    &--old {
-      @include var-color('color', $cx-search-list-price-old-color);
-      text-decoration: line-through;
-    }
+  &.old {
+    color: var(--cx-g-color-seconadry);
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-list/product-list-item/product-list-item.component.scss]: Typo in `--cx-g-color-secondary`.

File: projects/storefrontlib/src/lib/product/components/product-list/product-list-item/product-list-item.component.scss [hunk 2]
```diff
@@ -1,73 +1,51 @@
...
-    text-align: center;
+    border-width: var(--cx-border-width, 0 0 0 0);
   }
+}
 
-  &__image {
-    width: $cx-search-list-image-width;
-  }
+.cx-product-image-container {
+  display: var(--cx-display, block);
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-list/product-list-item/product-list-item.component.scss]: I don't think the display, text-align and width should be variables. I don't see a use case to override this.

File: projects/storefrontlib/src/lib/product/components/product-list/product-list-item/product-list-item.component.scss [hunk 3]
```diff
@@ -1,73 +1,51 @@
...
+}
 
-  &__price {
-    @include type('3');
-    text-align: left;
-    display: inline-block;
-    margin: $cx-search-list-price-trbl-margin;
+.cx-product-price {
+  @include type('3');
+  text-align: var(--cx-text-align, left);
```
Comment (LTiger14) [inline projects/storefrontlib/src/lib/product/components/product-list/product-list-item/product-list-item.component.scss]: I don't think this you be a variable you should just remove it if it's the default. In .cx-product-name also.

File: projects/storefrontlib/src/lib/product/components/product-list/product-list-item/product-list-item.component.scss (hunk 2 above)
Comment (LTiger14) [reply projects/storefrontlib/src/lib/product/components/product-list/product-list-item/product-list-item.component.scss]: Still not resolved

**Comment by developpeurweb** [comment]
Fixes GH-616

**Pull request merged**